frontend/src/pages/builder/components/Editor.tsx

```diff
@@ -147,7 +147,7 @@ export const Editor = ({
   const [readOnly] = useState<boolean>(editableProps?.readOnly || false)
   const plugins = getCommonPlugins()
   useEffect(() => {
-    setActiveEditorId(templateId || '')
+    setActiveEditorId(templateId || 'defaultEditor')
```

Comment:
This seems to break creating a new template: 

https://github.com/opengovsg/memos/blob/b3d459d32b2e247258c24a6accea141574c2ebe5/frontend/src/features/builder/EditorContext.tsx#L84-L86

We call the update template endpoint if `activeEditorId` is defined, so this change results in the update endpoint always being called instead of the save endpoint.